### DISCUSSION TITLE
is this project dead?

### DISCUSSION THREAD

**Discussion by n0rt0nthec4t** [containrrr/watchtower#2063] [General]
Is this project no longer maintained? Seems issues are starting to backup, no fixes, more bugs and nothing released for over 12months?

**Comment by Rundervinkje** [comment]
I guess that answers your question :).
On to: https://crazymax.dev/diun/ i guess then.

**Comment by mrmuiz** [comment]
Sounds dead to me. On to this one  i gues https://github.com/nicholas-fedor/watchtower/releases
